tests/shell_ble/tests/01-run.py

```diff
@@ -0,0 +1 @@
+../../shell/tests/01-run.py
```

Comment:
If this test is added it should be under `tests-with-config`

Reply:
Absolutely, I moved the test directory